External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -2627,8 +2627,13 @@ private static ASResult EvalExpression(string expression, ASExpr context, FileMo
         {
             ASResult notFound = new ASResult {Context = context};
             if (string.IsNullOrEmpty(expression)) return notFound;
-
             var ctx = ASContext.Context;
+            ClassModel type;
+            if (!string.IsNullOrEmpty(context.WordBefore))
```

Comment:
This kind of cases should have a little comment like `// case: delete a / typeof a

Reply:
and `cast`, and `untyped` and *OtherOperators*... )

Reply:
fixed